src/components/Details/Details.js

```diff
@@ -65,86 +67,53 @@ const Details = ({
 
   const handleEditInput = useCallback(
     (value, field) => {
-      detailsDispatch({
-        type: detailsActions.SET_CHANGES_DATA,
-        payload: {
-          ...detailsState.changes.data,
-          [field]: value
-        }
-      })
+      setChangesData({ [field]: value })
```

Comment:
if we already have some field in changes data and you'll set data this way, how do you think will we still have that field in data + new field or just a new field?